setup.cfg

```diff
@@ -78,7 +78,7 @@ testpaths = "jdaviz" "docs"
 astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
-addopts = --doctest-rst --import-mode=append
+addopts = --doctest-rst --import-mode=append -p no:warnings
```

Comment:
@pllim This is what I had to do to prevent `Spectrum1D reshaping flux array` warnings from running tests in CI. Please let me know if there is an alternative to this since it feels wrong!